packages/components/src/dropdown/index.tsx

```diff
@@ -42,12 +47,14 @@ export default function Dropdown( props ) {
 		onClose,
 		onToggle,
 		style,
-	} = props;
+	}: WordPressComponentProps< DropdownProps, 'div' >,
+	forwardedRef: ForwardedRef< any >
+) {
 	// Use internal state instead of a ref to make sure that the component
 	// re-renders when the popover's anchor updates.
 	const [ fallbackPopoverAnchor, setFallbackPopoverAnchor ] =
-		useState( null );
-	const containerRef = useRef();
+		useState< HTMLDivElement | null >( null );
+	const containerRef = useRef< HTMLDivElement | null >( null );
```

Comment:
This bottom one can just remain undefined, since it never took `null` to begin with:

```suggestion
	const containerRef = useRef< HTMLDivElement >();
```

Reply:
Thanks, committed in https://github.com/WordPress/gutenberg/pull/45787/commits/82bbf50f6e0f54353a58b69fc8bae3343b92c29b